lib/Http/Request/Resolver/TemplateResolver.php

```diff
@@ -30,7 +30,7 @@ public function getTemplate(Request $request = null): ?string
             $request = $this->getCurrentRequest();
         }
 
-        return $request->attributes->getString(DynamicRouter::CONTENT_TEMPLATE);
+        return $request->get(DynamicRouter::CONTENT_TEMPLATE);
```

Comment:
ops... i just went straightforward because i've noticed the setter below
https://github.com/pimcore/pimcore/blob/68f198d32add713f2e4025b5dcb6414754133145/lib/Http/Request/Resolver/TemplateResolver.php#L38

Using the `$request->get()` is not best practice anymore.
Should it be 
```suggestion
        return $request->query->get(DynamicRouter::CONTENT_TEMPLATE);
```
?

Or slightly better as
```suggestion
        return $request->query->get(DynamicRouter::CONTENT_TEMPLATE) ?? $request->attributes->get(DynamicRouter::CONTENT_TEMPLATE);
```
?

Reply:
I think it's not in `query` i think ` attributes` it is

Reply:
ahh, ok, so it's `nullable` and getString was returning empty string